app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -5,51 +5,57 @@ class AamvaStateMaintenanceWindow
     # All AAMVA maintenance windows are expressed in 'ET' (LG-14028),
     TZ = 'America/New_York'
 
+    Window = Data.define(:cron, :duration) do
+      def cron_val
```

Comment:
Minor: I'd generally prefer we avoid the abbreviation since I don't think it's worth the potential readability hit to save two characters.

```suggestion
      def cron_value
```